apps/slack/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.19.0
```

Comment:
I suggest we create two of these, one for the frontend and one for the backend.

The backend serverless.yml says we're running there on node14, so I'd suggest on the lambda folder we do a v14 to align our dev environment with the actual serverless environment

Reply:
Btw I see this conflicts with what they say in the README 🤔 .

But here's the link: https://github.com/contentful/apps/blob/master/apps/slack/lambda/serverless.yml#L14

So the danger here is you could introduce a feature only available in node 16 that works locally but fails in production in the node 14 environment.